auth/basic/src/main/java/org/trellisldp/auth/basic/BasicAuthFilter.java

```diff
@@ -32,13 +33,19 @@
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerRequestFilter;
 import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.ext.Provider;
+
+import org.slf4j.Logger;
 
 /**
  * A basic authentication filter using an Authorization HTTP header.
  */
+@Provider
 @Priority(AUTHENTICATION)
 public class BasicAuthFilter implements ContainerRequestFilter {
 
+    private static final Logger LOGGER = getLogger(BasicAuthFilter.class);
+    
```

Comment:
There's some trailing whitespace here, which the checkstyle config if complaining about.